lighthouse-core/audits/byte-efficiency/offscreen-images.js

```diff
@@ -134,8 +134,8 @@ class OffscreenImages extends ByteEfficiencyAudit {
       });
 
       const headings = [
-        {key: 'preview', itemType: 'thumbnail', text: ''},
-        {key: 'url', itemType: 'url', text: 'URL'},
+        {key: 'preview', itemType: 'thumbnail', text: '', styleClass: 'preview'},
+        {key: 'url', itemType: 'url', text: 'URL', styleClass: 'url'},
```

Comment:
alright talked with paul too, for now we'll just add `lh-table-column--${item.type}` and get rid of this `styleClass` business